core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -2158,9 +2158,9 @@ class WorkspaceService(protected val userInfo: UserInfo,
   // todo: update this line as part of https://broadworkbench.atlassian.net/browse/CA-1220
   // This is done sequentially intentionally in order to avoid conflict exceptions as a result of concurrent IAM updates.
     List(
-      billingProjectOwnerPolicyEmail -> Set(terraBillingProjectOwnerRole, terraWorkspaceCanComputeRole),
-      policyEmailsByName(SamWorkspacePolicyNames.owner) -> Set(terraWorkspaceCanComputeRole),
-      policyEmailsByName(SamWorkspacePolicyNames.canCompute) -> Set(terraWorkspaceCanComputeRole)
+      billingProjectOwnerPolicyEmail -> Set(terraBillingProjectOwnerRole, terraWorkspaceCanComputeRole, "roles/lifesciences.workflowsRunner"),
```

Comment:
Do we think this is the only role that NextFlow will need (and we won't need to add additional roles or remove this role in the future)?

If yes:
I think the current implementation is ok.

If no or not sure:
I don't have a lot of context around NextFlow, but if we expect that things may change, then it will be easier manage this permission (and other potential permissions for NextFlow) via a custom IAM role.

The benefits of using a custom IAM role, assuming we change roles for Nextflow in the future:
* Eliminates the need for a migration to update each workspace. Instead, we just need to update the custom IAM role once (per env). Adding or removing roles is much simpler this way.
* If there are multiple roles added for NextFlow, they are conveniently grouped within the custom IAM role.

Reply:
We talked offline yesterday and looks like it will be best to add a custom role. I merged these two PRs doing that and am working on integrating the new nextflow custom role with this rawls PR and that'll be the last step:
- https://github.com/broadinstitute/terraform-terra/pull/162 merged and ran the script in all three orgs with @andy7i 
- https://github.com/broadinstitute/firecloud-develop/pull/2924 merged and pulls new nextflow custom role into rawls conf